__tests__/TimelineResultCompTest.js

```diff
@@ -21,6 +21,17 @@ describe("TimelineResultComp", () => {
 	});
 
 	it("has a div element", function() {
+		expect(wrapper.find('div').length).toEqual(1);
+	});
+
+	it("has id 'timelinePlaceholder", function() {
+		expect(wrapper.find('#timelinePlaceholder').length).toEqual(1);
+	});
+
+	it("passes timelineResult, className props correctly", function () {
+		wrapper = mount(React.createElement(TimelineResultComp, {className: 'testClassName', timelineResult: 'testTimelineResult'}, null));
+		expect(wrapper.props().timelineResult).toEqual("testTimelineResult");
+		expect(wrapper.hasClass('testClassName')).toEqual(true);
```

Comment:
I must have not actually checked but you still have a mix of single and double quotes.